testing/src/main/java/org/oppia/testing/TestingUtils.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.testing
+
+import android.view.View
+import android.widget.EditText
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.matcher.ViewMatchers
+import org.hamcrest.CoreMatchers
+import org.hamcrest.Matcher
+
+/**
+ * Appends the specified text to a view. This is needed because Robolectric doesn't seem to
+ * properly input digits for text views using 'android:digits'. See
+ * https://github.com/robolectric/robolectric/issues/5110 for specifics.
+ */
+
+fun appendText(text: String, testCoroutineDispatchers: TestCoroutineDispatchers): ViewAction {
```

Comment:
How about if we create a particular class and keep this method inside that, and there we can also do constructor injection of `testCoroutineDispatchers`. Also, run the test cases in both espresso and robolectric to see if this works after your change, just to confirm things work fine. 
Take the example of `ProfileTestHelper`. 
In this way, if in future we get more such methods which are repetitive in nature, we can put them here.

Reply:
Okay, that is surely a good idea.